CONTRIBUTING.md

```diff
@@ -5,6 +5,58 @@ First of all, thanks for contributing!
 This document provides some basic guidelines for contributing to this repository.
 To propose improvements, feel free to submit a PR or open an Issue.
 
+## Setup your developer Environment
+
+To setup your enviroment, make sure you installed [Android Studio](https://developer.android.com/studio).
+
+**Note**: you can also compile and develop using only the Android SDK and your IDE of choice, e.g.: IntelliJ Idea, Vim, etc.
+
+### Modules
+
+This project hosts the following modules:
+
+  - `dd-sdk-android`: the main library implementing all Datadog features (Logs, Traces, RUM, Crash reports);
+  - `dd-sdk-android-timber`: a lightweight library providing a bridge integration between `dd-sdk-android` and [Timber](https://github.com/JakeWharton/timber);
+  - `dd-sdk-android-ktx`: a set of Kotlin extensions to make the `dd-sdk-android` library more Kotlin friendly;
+  - `instrumented/benchmark`: a test module to verify the performance of the library;
+  - `instrumented/integration`: a test module with integration tests using Espresso;
+  - `tools/detekt`: a few custom [Detekt](https://github.com/arturbosch/detekt) static analysis rules;
+  - `tools/noopfactory`: an annotation processor generating no-op implementation of interfaces;
+  - `tools/unit`: a utility library with code to help writing unit tests;
```

Comment:
```suggestion
  - `tools/unit`: an utility library with code to help writing unit tests;
```

Reply:
Nope, in english, it's "a utility" ;)
cf https://www.a-or-an.com/a_an/utility